### PR TITLE
Dev docs

### PR DESCRIPTION
**Pull request by e111077**
Added development guide

[link to preveiw](https://github.com/PolymerLabs/code-sample-editor/blob/76e32e32896930bfd6741b63b27bc67dcf0ff36e/docs/development.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,6 +171,24 @@ project layout:
...
 ```
 
+```html
+<!-- index.html -->
+<html>
+  <head>
+    <script type="module">
+      {
+        {
+          headScript;
```
Comment (usergenic) [inline README.md]: This formatting doesn't look right since I assumed the intent was `{{headscript}}`?

Comment (e111077) [reply README.md]: noooo autolinter!

Comment (e111077) [reply README.md]: fixed

**Comment by e111077** [comment]
ping-a-ling

**Comment by justinfagnani** [comment]
This is pretty old by now and out of sync with master. Let's close it. Thanks for the work on the docs @e111077 !

**Pull request closed without merging**